docs/posts/2022-05-24.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: Post
+title: Hello World 4
+date: 2022-05-24
+useHeaderImage: false
+headerImage: /img/in-post/2021-12-24/header.jpg
+headerMask: rgba(0, 0, 0, .4)
+catalog: false
+hide: false
+tags: [test]
+---
```

Comment:
Maybe I'm missing something, but what's the purpose of this file? 😂

Reply:
This file is to test the display effect when the title in the next button and previous button only occupies one line